db/Models/DocumentCategory.cs

```diff
@@ -14,6 +14,7 @@ public class DocumentCategory : EntityBase
     public const string ORDERS = "ORDERS";
     public const string PLEADINGS = "PLEADINGS";
     public const string PSR = "PSR";
+    public const string REPORT = "REPORT";
```

Comment:
This seems a bit confusing to me, as PSR, is a source code, that appears to be only used in the original PCSS response, and REPORT is a JASPER only code that is logically equivalent to PSR.

I wonder if we could keep all of the PCSS codes values as-is, bug just format the display/description how we need on the frontend? Perhaps I'm just no seeing the value of mapping the code values from PCSS to JASPER equivalents. Am I missing something?